checker/bin/mapreduce-checker.sh

```diff
@@ -0,0 +1,120 @@
+  We recommend users to set this number according to your Hadoop cluster size,
+  so that MapReduce integration checker can check more Hadoop nodes.
+  By default, the INPUT_SPLITS is 10.
+
+-h  display this help."
+
+ALLUXIO_PATH=$(cd "${BIN}/../../"; pwd)
+ALLUXIO_JAR_PATH=""
+HADOOP_LOCATION=""
+NUM_MAPS="${1:-10}";
+
+# Find the location of hadoop in order to trigger Hadoop job
+function find_hadoop_path() {
+  [ -f "$(which hadoop)" ] && HADOOP_LOCATION="$(which hadoop)"
+
+  # If ${HADOOP_HOME} has been set
+  if [[ "${HADOOP_LOCATION}" == "" ]] && [[ "${HADOOP_HOME}" != "" ]]; then
+    {
+      [ -f "${HADOOP_HOME}/hadoop" ] && HADOOP_LOCATION="${HADOOP_HOME}/hadoop"
+    } ||   {
+      [ -f "${HADOOP_HOME}/bin/hadoop" ] && HADOOP_LOCATION="${HADOOP_HOME}/bin/hadoop"
+    } || {
+      [ -f "${HADOOP_HOME}/libexec/bin/hadoop" ] && HADOOP_LOCATION="${HADOOP_HOME}/libexec/bin/hadoop"
+    } || { # Try to find hadoop executable file in HADOOP_HOME
+      if [[ "${HADOOP_LOCATION}" == "" ]]; then
+        array=(`find "${HADOOP_HOME}" -type f -name 'hadoop'`)
+        for i in "${array[@]}"; do
+          HADOOP_LOCATION="$i"
+          break;
+        done
+      fi
+    }
+  fi
+
+  if [[ "${HADOOP_LOCATION}" == "" ]]; then
+    if [[ "${HADOOP_HOME}" != "" ]]; then
+      echo -e "Cannot find executable file hadoop in your HADOOP_HOME: ${HADOOP_HOME}, please check your HADOOP_HOME." >&2
+    else
+      echo -e "Please set HADOOP_HOME before running MapReduce integration checker." >&2
+    fi
+    exit 1
+  fi
+}
+
+function trigger_mapreduce() {
+  # Without -libjars, we assume that the Alluxio client jar has already been distributed on the classpath of all Hadoop nodes
+  ${LAUNCHER} "${HADOOP_LOCATION}" jar "${BIN}/../target/alluxio-checker-${VERSION}-jar-with-dependencies.jar" \
+    alluxio.checker.MapReduceIntegrationChecker "${NUM_MAPS}"
+
+  # Use -libjars if the previous attempt failed because of unable to find Alluxio classes and add remind information
+  if [[ "$?" == 2 ]]; then
+    echo "Try to use -libjars command line option to help distribute Alluxio client jar." \
+      >> "./MapReduceIntegrationReport.txt"
+    ${LAUNCHER} export HADOOP_CLASSPATH="${ALLUXIO_JAR_PATH}"
+    ${LAUNCHER} "${HADOOP_LOCATION}" jar "${BIN}/../target/alluxio-checker-${VERSION}-jar-with-dependencies.jar" \
+      alluxio.checker.MapReduceIntegrationChecker -libjars "${ALLUXIO_JAR_PATH}" "${NUM_MAPS}"
+
+    if [[ "$?" == 0 ]]; then
+      echo "Please use the -libjars ${ALLUXIO_JAR_PATH} command line option when using hadoop jar." \
+        >> "./MapReduceIntegrationReport.txt"
+      echo "" >> "./MapReduceIntegrationReport.txt"
+      echo "Please export HADOOP_CLASSPATH=${ALLUXIO_JAR_PATH} to make Alluxio client jar available to client JVM created by hadoop jar command " \
+        >> "./MapReduceIntegrationReport.txt"
+    fi
+  fi
```

Comment:
The results now look like:
![image](https://user-images.githubusercontent.com/25273483/36236675-326e134a-11ac-11e8-8af0-0590cc04cfe1.png)

Reply:
We should add something in the middle along the lines of "Re-running integration checker with HADOOP_CLASSPATH=${HADOOP_CLASSPATH}/path/to/client and -libjars /path/to/client"? Otherwise the user will be confused about why the checker ran twice and had different results.